actions/check_clarity_contacts_in_supr.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python
+
+from st2actions.runners.pythonrunner import Action
+
+from genologics.lims import *
+from genologics.config import BASEURI, USERNAME, PASSWORD
+from lib.supr_utils import * 
+
+#disable ssl warnings
+import requests.packages.urllib3
+requests.packages.urllib3.disable_warnings()
+
+
+class CheckClarityContactsInSupr(Action):
+    """
+    Retrives contacts from open projects in ClarityLIMS and checks for
+    SUPR accounts by email.
+    Then composes an email body to notify project coordinators.
+    """
+
+    def fetch_open_projects(self):
+        lims = Lims(BASEURI, USERNAME, PASSWORD)
+        lims.check_version()
+        projects = lims.get_projects() 
+	filtered_projects = list()
+	for project in projects:
+		if project.open_date and not project.close_date:
+			filtered_projects.append(project)
+        return filtered_projects
+
+    def check_email_in_supr(self, email_adress):
+        email_missing = False
+        email_multi = False
+
+        try:
+            supr_id = SuprUtils.search_by_email(self.supr_api_url, email_adress, self.supr_api_user, self.supr_api_key)
+        except AssertionError as ae:
```

Comment:
I may be wrong but I was under the impression that assertions shouldn't be used for business logic as they are more or less a debugging tool.